lightwood/api/generate_config.py

```diff
@@ -145,10 +145,11 @@ def generate_config(type_information: TypeInformation, statistical_analysis: Sta
         timeseries_transformer = {
             'object': 'transform_timeseries',
             'config_args': {
-                'timeseries_settings': 'problem_definition.timeseries_settings'
+                'problem_definition': 'problem_definition'
```

Comment:
Could you request the particular members you need and not the whole problem defintion, in line with keeping coupling as minimal as possible. Ideally we should aim for requesting no single value that is never used.

Reply:
Note: I know the rest of lightwood isn't perfect at this right now either, but I think it's the goal we should work towards.